core/classes/Token.php

```diff
@@ -36,7 +36,10 @@ public static function get() {
     // Check a token in session matches
     // Params: $token (string) - contains the form token which will be checked against the session variable
     public static function check($token = null) {
-        if ($token == null) $token = Input::get('token');
```

Comment:
I don't think it was necessary to add {}.

Reply:
https://stackoverflow.com/questions/359732/why-is-it-considered-a-bad-practice-to-omit-curly-braces